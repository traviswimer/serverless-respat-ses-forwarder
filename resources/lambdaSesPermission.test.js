const lambdaSesPermission = require('./lambdaSesPermission');

describe('lambdaSesPermission()', () => {
	test('generates permission', () => {
		expect(lambdaSesPermission({
			prefix: "ThePrefix",
			pattern_name: "ThePatternName",
			lambda_function_name: "TheLambdaName"
		})).toMatchSnapshot();
	});
});
