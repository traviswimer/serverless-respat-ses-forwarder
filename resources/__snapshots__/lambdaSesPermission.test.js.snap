// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lambdaSesPermission() generates permission 1`] = `
Object {
  "Properties": Object {
    "Action": "lambda:InvokeFunction",
    "FunctionName": Object {
      "Fn::GetAtt": Array [
        "TheLambdaNameLambdaFunction",
        "Arn",
      ],
    },
    "Principal": "ses.amazonaws.com",
  },
  "Type": "AWS::Lambda::Permission",
}
`;
